### PR TITLE
Events tab bar changes

### PR DESCRIPTION
**Pull request by josephriosIO**
## What was changed
Made the tab bar look more like new UI without the pieces we don't have yet

# Images
Screen Shot 2021-12-02 at 3 18 06 PM

### CODE REVIEW

File: tailwind.config.cjs [hunk 1]
```diff
@@ -5,6 +5,8 @@ const TemporalColors = {
   secondary: '#64748b',
   // Custom Colors
   primaryText: '#141414',
+  historybg: '#DCE4FB',
```
Comment (stevekinney) [inline tailwind.config.cjs:8]: Do we want to capitalize the `b`?

**Comment by stevekinney** [comment]
I love it… I just have the tiniest nit.

File: tailwind.config.cjs (hunk 1 above)
Comment (josephriosIO) [reply tailwind.config.cjs:8]: done and done

File: tailwind.config.cjs [hunk 2]
```diff
@@ -5,6 +5,8 @@ const TemporalColors = {
   secondary: '#64748b',
   // Custom Colors
   primaryText: '#141414',
+  historyBg: '#DCE4FB',
```
Comment (stevekinney) [inline tailwind.config.cjs:8]: Do we want to have a token dedicated to something so small? /cc @GiantRobots

Comment (canvanooo) [reply tailwind.config.cjs:8]: It's technically the same as the Running status badge. We do use that pattern elsewhere..

Comment (josephriosIO) [reply tailwind.config.cjs:8]: I can rename it or we can move it to just in a style sheet

File: src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/_tabs.svelte [hunk 3]
```diff
@@ -1,29 +1,38 @@
...
+    @apply block;
   }
 
   a.active {
-    @apply border-b-8 border-purple-400 font-bold;
+    @apply border-b-2 rounded-b-sm border-primary font-medium;
+  }
+
+  span {
+    background-color: #dce4fb;
```
Comment (stevekinney) [inline src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/_tabs.svelte:35]: @canvanooo Should we just use `blue-700` and `blue-100` here?

**Pull request merged**